tutorials/tts/TTS_pronounciation_customization.ipynb

```diff
@@ -0,0 +1,290 @@
+    "# How do I customize TTS pronunciations?\n",
+    "\n",
+    "This tutorial walks you through the basics of NeMo TTS pronunciation customization. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can either run this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "\"\"\"\n",
+    "\n",
+    "BRANCH = 'main'\n",
+    "# # If you're using Google Colab and not running locally, uncomment and run this cell.\n",
+    "# !apt-get install sox libsndfile1 ffmpeg\n",
+    "# !pip install wget text-unidecode pynini==2.1.4\n",
+    "# !python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[all]\n",
+    "# !wget https://raw.githubusercontent.com/NVIDIA/NeMo/main/nemo_text_processing/install_pynini.sh\n",
+    "# !bash install_pynini.sh"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Grapheme-to-phoneme (G2P) Overview\n",
+    "\n",
+    "Modern **text-to-speech** (TTS) models can learn pronunciations from raw text input and its corresponding audio data.\n",
+    "Sometimes, however, it is desirable to customize pronunciations, for example, for domain-specific terms. As a result, many TTS systems use grapheme and phonetic input during training to directly access and correct pronunciations at inference time.\n",
+    "\n",
+    "\n",
+    "[The International Phonetic Alphabet (IPA)](https://en.wikipedia.org/wiki/International_Phonetic_Alphabet) and [ARPABET](https://en.wikipedia.org/wiki/ARPABET) are the most common phonetic alphabets. \n",
+    "\n",
+    "There are two ways to customize pronunciations:\n",
+    "\n",
+    "1. pass word's phonetic as an input to the TTS model, note that the request-time overrides are best suited for one-off adjustments\n",
+    "2. configure TTS model with the desired domain-specific terms using custom phonetic dictionary\n",
+    "\n",
+    "Both methods require users to convert graphemes into phonemes (G2P). \n",
+    "\n",
+    "#### All words for G2P purposes could be divided into the following groups:\n",
+    "* *known* words - words that are present in the model's phonetic dictionary\n",
+    "* *out-of-vocabulary (OOV)* words - words that are missing from the model's phonetic dictionary. \n",
+    "* *[heteronyms](https://en.wikipedia.org/wiki/Heteronym_(linguistics)* - words with the same spelling but different pronunciations and/or meanings, e.g., *bass* (the fish) and *bass* (the musical instrument).\n",
```

Comment:
This link is mildly broken: it cuts off at `..._(linguistics` without the final paren.